iOS-School-02_stack.playground/Contents.swift

```diff
@@ -0,0 +1,65 @@
+// MARK: - Stack
+/// Standard functions for working with the stack.
+class Stack {
+    private var stackElements = [Int]()
+
+    /// Adds an element the top of the stack.
+    /// - Parameter element: The number `Int` to add at the top of the stack
```

Comment:
If you have decided to have dot signs (`.`) then use it consistently:
```suggestion
    /// - Parameter element: The number `Int` to add at the top of the stack.
```

Reply:
Fixed.